libwallet/src/api_impl/owner_swap.rs

```diff
@@ -1407,7 +1682,7 @@ where
 					&swap.electrum_node_uri1,
 					&swap.electrum_node_uri2,
 				)?;
-				let swap_api = crate::swap::api::create_instance(
+				let swap_api = crate::swap::api::create_btc_instance(
```

Comment:
Please add support for eth as well.  in case of fail_bidding message, we need to cancel the trades.  Other party will not continue with trade.

Reply:
swap market will be support after atomicswap